PENDING.md

```diff
@@ -38,6 +38,8 @@ BREAKING CHANGES
 * [types] sdk.NewCoin now takes sdk.Int, sdk.NewInt64Coin takes int64
 * [cli] #1551: Officially removed `--name` from CLI commands
 * [cli] Genesis/key creation (`init`) now supports user-provided key passwords
+* [cli] --print-response now defaults to true in commands that create and send a transaction
+* [cli] you can now pass --public-key or --address to `gaiacli keys show` to return a plaintext representation of the key's address or public key for use with other commands
```

Comment:
I think pubkey is generally accepted abbreviation, so maybe rename to `--pubkey`?

Reply:
++

@mslipper may you also reference the PR here?